core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -127,7 +127,7 @@ const setRole = async function(browser, page, role) {
       ').some(' +
       '  (statusMessageElem) => statusMessageElem.innerText.includes(' +
       '    "' + role + '"))', {
-        polling: 500
+        polling: 250
```

Comment:
Can you show some evidence that this fixes the flake? Maybe the successful log of a few successive runs? Or please justify why.

Reply:
@nithusha21 There is a justification in the description of the PR -- I believe the lighthouse tests aren't checking the `statusMessage` element fast enough. By decreasing `polling` to 250ms, I am making the test check the element more frequently, thus reducing the chance that it misses the `statusMessage`

I've rerun the lighthouse accessibility tests 6 times on this PR and they haven't failed yet!